README.md

```diff
@@ -273,6 +314,15 @@ environments:
       publishing:
         - 80
 ```
+#### AWS Command Line Access
+
+Follow the guide at Configuring the AWS CLI to use AWS Single Sign-On to set up CLI access with SSO. The details needed are:
```

Comment:
Where is the "Configuring the AWS CLI" guide? This should be linked to from here

Reply:
yep forgot to attach the link